### DISCUSSION TITLE
XML parsing error - how to fix it?

### DISCUSSION THREAD

**Discussion by rpatraquim** [Ifsttar/I-Simpa#316] [Q&A]
Suddenly I got this error: "XML parsing error: 'unclosed token' at line 7". I-SIMPA won't even start... How can I fix it?

![imagem](https://user-images.githubusercontent.com/102927347/162330502-a1f01142-1d4d-435d-8c62-9f02bc88d352.png)

**Comment by nicolas-f** [comment]
Hi
The current project xml seem to be corrupted. You have to delete the I-Simpa directory here:
%APPDATA%\roaming\isimpa

Something like that. If you not find it just search isimpa into the %APPDATA% folder

**Reply by rpatraquim** [reply]
Thank you very much Nicolas!!! It is all right now!!! Solved!

P.S. - by the way, i´ve got more questions about I-Simpa results, but won't put them here!

**Reply by Pierre-Moulin** [reply]
Hello, I have quite the same problem but it occur with the importation of the scene 
The soft is fixed on it ...  even if I uninstall/install the soft. 
Any idea :/ ?
![image](https://user-images.githubusercontent.com/106664450/185862644-60f840bc-b168-4429-b10c-3feca10f7b9e.png)
![image](https://user-images.githubusercontent.com/106664450/185863139-8568ce56-592a-4c6d-ad47-0b46118a3855.png)

**Reply by Pierre-Moulin** [reply]
It is fine, my problem is resolve :)
